tests/code_size/hello_webgl_wasm2js.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 588,
   "a.html.gz": 386,
-  "a.js": 20852,
-  "a.js.gz": 8105,
+  "a.js": 20785,
```

Comment:
i'm confused, how was a win of this size not causing an error on the test? it's larger than the slop factor.

Reply:
IIUC, it *is* causing an error on the test..   we only run these tests on CI so that they don't show up until after the roll, right?

Reply:
Oh, it was erroring on CI here on github? I've been too busy to notice :smile: sgtm

Reply:
(yes, we don't test these on chromium CI, only here)